client/src/Login.js

```diff
@@ -7,7 +7,7 @@ import {
   Typography,
   Button,
   FormControl,
-  TextField,
+  TextField
```

Comment:
In future PRs, it is best if you only include files and changes that are directly related to the issue - this allows us to more better review your changes in isolation. This file is just formatting changes and updating deprecated props and does not need to be included. 🙂

Reply:
is it ok for me to leave it there for future PRs, or do you want me to revert it back to its original file ?